docs/github.md

```diff
@@ -0,0 +1,107 @@
+# Using Jujutsu with GitHub Projects
+
+This guide assumes a basic understanding of either Git or Mercurial. 
+
+## Setup an SSH key
+As of December 2022 it's recommended to set up a ssh key to work with Github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see [issue #469][http-auth] for more information and progress on authenticated http. 
+
+
+## Basic workflows
+You can either create a branch and then commit revisions on top of it (similiar 
+to how Git is typically used) or just start making changes in the repository
+(which is how Mercurial is used). 
+This allows you to continue using your favorite way of using the VCS. 
```

Comment:
I find this a bit confusing, because the workflow is the same whether or not you create a branch first. It's also odd to repeatedly mention creating a branch before working when a git user's intention is generally going to be to place the new commits, not their starting point, on the branch.

Reply:
I think to explain it better, you might have to mention co-location of branches and whether you want to work on two separate things at once (switch between branches) or not (since you can't see them both at once with Git).

Reply:
I agree. I had a long unfinished message here with a suggestion, but that seem to have gotten lost (probably by a reboot) :(

I would suggest describing the workflow without providing different ones for Git and Mercurial because I wouldn't recommend the Git-like one anyway since it only works for a single commit. Something like this:


```shell
# Start a new commit off of `main`
$ jj new main
# Refactor some files, then add a description and start a new commit
$ jj commit -m 'refactor(foo): restructure foo()'
# Add a feature, then add a description and start a new commit
$ jj commit -m 'feat(bar): add support for bar'
# Create a branch so we can push it to GitHub
$ jj branch create bar -r @-
# Push the branch to GitHub (pushes only `bar`)
$ jj git push
```